zstash/create.py

```diff
@@ -78,13 +74,16 @@ def create():
     logger.debug("Running zstash create")
     logger.debug("Local path : %s" % (config.path))
     logger.debug("HPSS path  : %s" % (config.hpss))
-    logger.debug("Max size  : %i" % (config.maxsize))  # type: ignore
+    logger.debug("Max size  : %i" % (config.maxsize))
     logger.debug("Keep local tar files  : %s" % (config.keep))
 
     # Make sure input path exists and is a directory
     logger.debug("Making sure input path exists and is a directory")
-    # FIXME: Argument 1 to "isdir" has incompatible type "None"; expected "Union[str, bytes, _PathLike[str], _PathLike[bytes]]"mypy(error)
-    if not os.path.isdir(config.path):  # type: ignore
+    if config.path:
+        path = config.path
+    else:
+        raise Exception("Invalid config.path={}".format(config.path))
```

Comment:
You should use raise ValueError(...) to be more explicit on what the exception is.

https://docs.python.org/3/library/exceptions.html#ValueError
https://stackoverflow.com/questions/43605512/python-difference-between-valueerror-and-exception